packages/kbn-zod-helpers/kibana.jsonc

```diff
@@ -1,6 +1,10 @@
 {
-  "devOnly": false,
+  "type": "shared-common",
   "id": "@kbn/zod-helpers",
-  "owner": "@elastic/security-detection-rule-management",
-  "type": "shared-common"
-}
+  "owner": [
+    "@elastic/security-detection-rule-management"
+  ],
+  "group": "platform",
+  "visibility": "shared",
+  "devOnly": false
```

Comment:
Do we have dev docs explaining these properties?

Reply:
That's a good point, we have dev docs for the [package types](https://docs.elastic.dev/kibana-dev-docs/ops/packages#package-types), but the rest of the properties are not covered.
I will add documentation for `group` and `visibility`.
`devOnly` is not in the scope of this PR, we can ask @elastic/kibana-operations if it is documented elsewhere.